about/package-scope.md

```diff
@@ -195,6 +195,27 @@ Packages to aid with instruction.
 
 ## Package technical scope
 
+
```

Comment:
from @tupui 
I think we should do something similar, provide a template basically and tell how/where the info should be presented to the users

****
leah: i really LOVE this idea of pyos having a badge on its package listing so users know what is being collected IF ANYTHING? that way we are super clear about that in our catalog so it's a value added element. We can also ask maintainers to add a badge / template to their README file and some standard language in there as well.